docker/package/scripts/udev-rules

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2018-2021 Ledger <https://www.ledger.com>
+#
+# SPDX-License-Identifier: LicenseRef-Apache-Ledger
+
```

Comment:
I think it would be worth putting in a comment here the repository and commit where this came from, for future reference.

Also I noticed that compared to a previous diff you removed the 
```
if [ ! -f /etc/udev/rules.d/20-hw1.rules ]; then
```
initial check. Did you find it to be unnecessary?

Overriding the udev config file is not an issue, but perhaps we'd want to avoid making the two `udevadm` calls if possible?

Reply:
I've decided to remove `if [ ! -f /etc/udev/rules.d/20-hw1.rules ]; then` in case we'll have to update provided udev rules in future.

Reply:
> we'd want to avoid making the two `udevadm` calls if possible

AFAIU, `udevadm` has to be called in case rules were updated, we can avoid calling it by looking at the diff between existing rules and rules that we're going to write, but the overhead for calling `udevadm` doesn't look significant for me

Reply:
:ok_hand: sounds reasonable to me

Reply:
Oh but I still think this:

> I think it would be worth putting in a comment here the repository and commit where this came from, for future reference.